docs/blog/newsletter-september-2024.rst

```diff
@@ -0,0 +1,115 @@
+:og:image: https://www.writethedocs.org/_static/logo-opengraph.png
+
+.. post:: September 04, 2024
+  :tags: newsletter
+
+##########################################
+Write the Docs Newsletter – September 2024
+##########################################
+
+Howdy, documentarians. I hope everything is going well in your part of the world. We're back from out regular August break and already setting records. So far, it's just heat records here, but maybe we'll set the record for most documentarians inspired this month.
+
+In conference news, the Atlantic conference is coming up this month on 22–24 September. It's virtual so there's still time to get a ticket and you don't have to worry about travel. See the `planned talks </conf/atlantic/2024/speakers/>`__ and then `grab a ticket </conf/atlantic/2024/tickets/>`__. And if seeing the list of talks gets you more inspired, the deadline for `talk proposals for the Australia conference </conf/australia/2024/cfp/>`__ is this Friday (6 September). So get your ideas in while you still can!
```

Comment:
@CollierCZ can you add a line at the end that says, Write the Docs Australia talks will be announced by end of September so folks can start planning travel etc?